Makefile

```diff
@@ -158,4 +158,32 @@ local-uninstall-chrome:
 
 local-uninstall: local-uninstall-firefox local-uninstall-chrome
 
-.PHONY: clean release local-install-firefox local-install-chrome local-install install local-uninstall-firefox local-uninstall-chrome local-uninstall uninstall
+PV = $(shell echo $(RELEASE_TAG) | sed -e s:^v::)
```

Comment:
Can we prefix these variables with `DEBIAN_` to avoid clashes with other variables?